src/views/FAQ.vue

```diff
@@ -9,17 +9,16 @@
         <hr width="30%">
         <p>
           if you can't find the answer of you question below visit our
-          <router-link
-            to="/create-profile"
-          >
+          <router-link to="/create-profile">
             Tutorial Page
-          </router-link>
-          to<br> receive more information about using Superhero platform and wallet.
+          </router-link>to
+          <br>receive more information about using Superhero platform and wallet.
         </p>
         <expandable-block :title="'What is Superhero?'">
-          Superhero is a P2P (Peer-to-Peer) platform for social sharing.<br>
-          The mission of Superhero is to empowering people to support people,
-          without intermediaries.<br>
+          Superhero is a P2P (Peer-to-Peer) platform for social sharing.
+          <br>The mission of Superhero is to empowering people to support people,
```

Comment:
imho, this should be
```suggestion
          <br>The mission of Superhero is to empower people to support people,
```

Reply:
I agree. These were the texts provided by the comms. Wasnt expecting to have an error there but 🤷‍♀ ..